Synapse3.SynapseModule/Patches/PlayerPatches/DeathPatches.cs

```diff
@@ -1,85 +1,109 @@
-﻿using System;
-using System.Linq;
-using HarmonyLib;
+﻿using HarmonyLib;
 using MEC;
 using Neuron.Core.Logging;
 using PlayerStatsSystem;
+using Synapse3.SynapseModule.Config;
 using Synapse3.SynapseModule.Dummy;
 using Synapse3.SynapseModule.Enums;
 using Synapse3.SynapseModule.Events;
 using Synapse3.SynapseModule.Player;
 using Synapse3.SynapseModule.Role;
+using System;
+using System.Linq;
+using System.Reflection;
 
 namespace Synapse3.SynapseModule.Patches.PlayerPatches;
 
 [Patches]
 [HarmonyPatch]
 internal static class DeathPatches
 {
+    static SynapseTranslation _translation;
+    static PlayerService _player;
+
+    static DeathPatches()
```

Comment:
Just read the official docs it is called upon first use of the class just like you said, but it could still be to early and cause problems later so we need to test it.

Reply:
it does as expected, the reflection has no impact on the call of the ctor. He is only called when the patch is called.

Reply:
Code of the ctor:
```cs
        NeuronLogger.For<Synapse>().Warn("Sy3 Event: Player Death Event Ctor\n");

        _translation = Synapse.Get<SynapseConfigService>().Translation;
        _player = Synapse.Get<PlayerService>();

        NeuronLogger.For<Synapse>().Warn($"Sy3 Event: {_player == null}\n");
````
Call after the log of damage and not last the creation of Synapse.
![log](https://user-images.githubusercontent.com/64769541/203380911-3867d401-0703-4a46-a76f-6224d183070f.png)